spt/strafestuff.hpp

```diff
@@ -322,7 +322,18 @@ bool Strafe(PlayerData& player, const MovementVars& vars, bool onground, bool ju
 {
 	//DevMsg("[Strafing] ducking = %d\n", (int)ducking);
 	if (jumped && player.Velocity.Length2D() >= vars.Maxspeed * ((ducking || (vars.Maxspeed == 320)) ? 0.1 : 0.5)) {
-		out.Yaw = NormalizeDeg(tas_strafe_yaw.GetFloat() + 180);
+		if tas_strafe_glitchless.GetBool() == 1 {
```

Comment:
1. Paranthesis are required, so `if (condition)`
2. No need for quality check here, just `if (tar_strafe_glitchless.GetBool())`